.travis.yml

```diff
@@ -10,10 +10,14 @@ python:
   - "2.7"
 
 # command to install dependencies
-install: "pip install -Ue .[test,build]"
+install: 
+  - "pip install -Ue .[test,build]"
+  - pip install pep8==1.5.7
 
 # command to run tests, e.g. python setup.py test
-script: nosetests --with-cov --cov=testfixtures
+script:
+  - nosetests --with-cov --cov=testfixtures
+  - pep8
```

Comment:
I've read it! I don't actually like all of PEP8, although with time I find that even the stuff I used not to like makes sense and I prefer to take it all. 
It makes your code easier to read for usual Python contributors, they don't have to guess what you want code to look like (it's actually one of those things for which Python is great, I feel) and it limits discussions of this nature.
In the end, it's your repo and my fork is there, so you can pull it and amend the commit to ditch the stuff you don't like.